core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java

```diff
@@ -29,7 +28,7 @@
 
     @Substitute
     public static Logger getLogger(Class<?> clazz) {
-        return LoggerFactory.getLogger(clazz.getName());
+        return Logger.getLogger(clazz.getName());
```

Comment:
This is definitely incorrect.  The substitution is meant to return an SLF4J logger, not a JBoss logger.

Reply:
Oops, i'll fix that.

Reply:
@dmlloyd , what do you think about choosing single Logger name convention in Quarkus ?

```java
private static final Logger LOGGER logger  LOG  log // which form do you suggest ?
```

Reply:
I don't have a very strong opinion, but I use `log` or `LOG` myself.